queries/zig/folds.scm

```diff
@@ -1,14 +1,22 @@
 [
-  (Block)
-  (ContainerDecl)
-  (SwitchExpr)
-  (InitList)
-  (AsmExpr)
-  (ErrorSetDecl)
-  (LINESTRING)
-  [
-    (IfPrefix)
-    (WhilePrefix)
-    (ForPrefix)
-  ]
+  (block)
+  (switch_expression)
+  (initializer_list)
+  (asm_expression)
+  (multiline_string)
+  (if_statement)
+  (while_statement)
+  (for_statement)
+  (if_expression)
+  (else_clause)
+  (for_expression)
+  (while_expression)
+  (if_type_expression)
+  (function_signature)
+  (parameters)
```

Comment:
Maybe `(call_expression)` here as well? Or too much

Reply:
why not